src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/GrpcService-CSharp.Client/Program.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Greet;
+using Grpc.Core;
+
+namespace Company.ClientApplication
+{
+    public class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var channel = new Channel("localhost:50051", ChannelCredentials.Insecure);
```

Comment:
We never came up with a solution for the client port being hardcoded?

What about if we attempt to get the port from the args and fallback to 50051 if it is not supplied.

```cs
var port = args.Length > 0 ? args[0] : null;
var channel = new Channel("localhost:" + port ?? "50051", ChannelCredentials.Insecure);
```

Or if that is a little heavy for a sample, we could set the port to a variable and have a comment prompting the dev to update it to match the local server port:

```cs
var port = "50051"; // Update to match the port of the gRPC server

var channel = new Channel("localhost:" + port, ChannelCredentials.Insecure);
```

Reply:
I'm in favor of option 2. Just add a comment